lib/matplotlib/patches.py

```diff
@@ -3678,6 +3678,377 @@ def __init__(self, widthB=1., lengthB=0.2, angleB=None):
 
     _style_list["-["] = BracketB
 
+    class _Brace(_Base):
+        """ Class used as a basis for arrowstyles involving brace-shaped head
+            or tail.
+        """
+        def __init__(self, braceA=None, braceB=None,
+                     bracetypeA='curly', bracetypeB='curly',
+                     widthA=1., widthB=1.,
+                     lengthA=0.4, lengthB=0.4,
+                     angleA=None, angleB=None,
+                     scaleA=None, scaleB=None):
+            self.braceA, self.braceB = braceA, braceB
+            self.bracetypeA, self.bracetypeB = bracetypeA, bracetypeB
+            self.widthA, self.widthB = widthA, widthB
+            self.lengthA, self.lengthB = lengthA, lengthB
+            self.angleA, self.angleB = angleA, angleB
+            self.scaleA, self.scaleB = scaleA, scaleB
+
+        # Declare it as a static method as it is also planned to be use to build
+        # a new type of patch that may be useful to annotate range of data.
+        # However, not sure the current arguments are the best suited for
+        # that purpose. (Might be better to directly pass "length," p0 and p6.)
+        #
+        # Would it be better to define a module-level '_get_brace' function?
+        #
+        @staticmethod
+        def _get_brace(x, y, width, length, angle, bracetype):
+            """
+            *angle*
+                rotation in degrees (anti-clockwise)
+
+            *bracetype*
+                'curly': a real curly bracket }, with rounded corners made of 
+                         quadratic Bezier curves
+                'straight': a simplified }, with the 7 control points p0...p6
+                            simply linked with straight lines.
+
+            Details of the basic design:
```

Comment:
Can you put two colons at the end of this line so the ASCII art gets rendered correctly in the docs.